server/vulnerabilities/cve_test.go

```diff
@@ -61,6 +61,18 @@ func (d *threadSafeDSMock) InsertCVEForCPE(ctx context.Context, cve string, cpes
 	return d.Store.InsertCVEForCPE(ctx, cve, cpes)
 }
 
+func withNetRetry(t *testing.T, fn func() error) error {
+	for {
```

Comment:
I'm guessing we don't need to add a max number of retries guard here, because the test will timeout after 30 seconds?

Reply:
Also maybe a better place for this would be `vulnerabilities/testing_utils.go`?

Reply:
> I'm guessing we don't need to add a max number of retries guard here, because the test will timeout after 30 seconds?

Correct. We set the `-timeout` flag on CI (1h when race enabled, 30m when disabled).

> Also maybe a better place for this would be vulnerabilities/testing_utils.go?

Mhm... Probably better to  move it to `nettest` (`withNetRetry` is not functionality we need on the `vulnerabilities` package). This is some tech debt we have: `vulnerabilities/testing_utils.go` and the like should be moved from the `vulnerabilities` package into the `_test.go` files or its own `vulnerabilitytest` package.